distrobox-create

```diff
@@ -511,12 +520,18 @@ generate_command() {
 	result_command="${result_command} ${container_manager_additional_flags}"
 
 	# Now execute the entrypoint, refer to `distrobox-init -h` for instructions
+	if [ -n "${container_pre_init_hook}" ]; then
+		pre_init_hooks_arg="--pre-init-hooks \"${container_pre_init_hook}\""
+	else
+		pre_init_hooks_arg=""
+	fi
 	result_command="${result_command} ${container_image}
 		/usr/bin/entrypoint -v --name \"${container_user_name}\"
 		--user ${container_user_uid}
 		--group ${container_user_gid}
 		--home \"${container_user_custom_home:-"${container_user_home}"}\"
 		--init \"${init}\"
+		${pre_init_hooks_arg}
```

Comment:
and here we just use `${container_pre_init_hook}`

Reply:
done!